content/blog/2020-12-24-darktable-3.4/2020-12-24-darktable-3.4.md

```diff
@@ -0,0 +1,182 @@
+
+- The _channel mixer_ module is superseded by the _color calibration_ module.
+- The _invert_ module is superseded by the _negadoctor_ module.
+- The _fill light_ and _zone system_ modules are now superseded by the _tone equalizer_ module.
+- The _global tonemap_ and _tone mapping_ modules are superseded by the _filmic rgb_ and _local contrast_ modules
+
+# Renamed Modules
+
+The following modules have been renamed to better describe their use. These modules can still be found using their old names in the module search box.
+
+- _denoise (non-local means)_ is now _astrophoto denoise_
+- _denoise (bilateral filter)_ is now _surface blur_
+
+
+# UI Enhancements
+
+Many existing modules have had updates to their UI
+
+- The _retouch_ and _white balance_ module layouts have been completely rewritten
+- It is now possible to choose different UI layouts for the _color balance_ module, in order to reduce its height
+- The _tone curve_ and _base curve_ modules have been aligned to use a common interface
+- A number of utility modules can now have their height adjusted by hovering your mouse over them and pressing _Ctrl+scroll_. 
+- The darkroom _history stack_ module now shows the changes that have been made between history items when you hover the mouse over the module
+- The _global color picker_ module UI has been overhauled
+- It is now possible to automatically hide the header buttons on processing modules in the darkroom
+- Hovering your mouse over the header of a processing module now displays a  tooltip providing detailed in-app documentation.
+
+# Other Changes
+
+- The tethering view has been reworked to improve stability and now includes the histogram module again
+- A new button has been added to enable focus peaking mode, to complement the existing keyboard shortcut
+- Clicking on the darkroom history stack module's reset button provides a one-click way to discard history on your current image
+- The history stack can now be truncated -- removing all history items above the selected one but without compressing -- in the darkroom by holding _Ctrl_ while clicking the "compress history stack" button
+- New functionality has been added to `darktable-cli`
+- The filter categories combobox in the _collect images_ are now organized into groups
+- The data and library databases are now regularly checked for corruption
+- Backups of the data and library databases are now taken automatically allowing you to easily restore to an earlier state in case of corruption
+- By default the copy and paste buttons now exclude some modules that can cuase issues when pasted between images. You can still include these modules by using the "copy parts" and "paste parts" buttons
```

Comment:
Only the copy excludes some module, I don't think we need to talk about paste here not "paste parts" at end of paragraph.

Reply:
Agreed - there's also a spelling mistake here: "cuase"

Reply:
Fixed these two things.